src/reports/components/instance-details.scss

```diff
@@ -51,80 +51,66 @@
-    }
-
-    .row:last-child {
-        border-bottom: none;
-    }
-}
-
-.report-instance-table .label {
-    font-size: 14px;
-    line-height: 20px;
-    font-family: $semiBoldFontFamily;
-    color: $primary-text;
-    text-align: left;
-    width: 70px;
-}
-
-.report-instance-table .instance-list-row-content {
-    color: $secondary-text;
-    padding: 12px 20px;
-    font-size: 14px;
-    line-height: 20px;
-    align-items: flex-end;
-    display: flex;
-}
-
-.report-instance-table .instance-list-row-content.content-snipppet {
-    font-family: Consolas, Segoe UI, Courier New, monospace;
-    white-space: normal;
-    word-wrap: break-word;
-    word-break: break-word;
-}
-
-table.report-instance-table {
     border-collapse: collapse;
-    overflow-x: auto;
-    word-break: break-word;
-}
 
-@media screen and (max-width: 480px) {
-    .report-instance-table tr {
-        display: block;
-    }
-
-    .report-instance-table td {
-        display: block;
+    overflow-wrap: break-word;
+    word-break: break-word;
 
-        .label {
-            text-align: left;
+    .row {
+        display: flex;
+        flex-direction: row;
+        flex-wrap: wrap;
+
+        // The padding on the parent and margins on the children are coupled; they
+        // are intended to produce the desired spacing regardless of whether the children
+        // are presented side-by-side or wrapped
+        padding-top: 2px;
+        padding-bottom: 14px;
+        padding-left: 0px;
+        padding-right: 20px;
+        > * {
+            margin-top: 12px;
+            margin-bottom: 0px;
+            margin-left: 20px;
+            margin-right: 0px;
+            padding: 0;
         }
 
-        .instance-list-row-content {
-            text-align: right;
+        border-bottom: 0.5px solid $neutral-10;
+        &:last-child {
+            border-bottom: none;
         }
     }
 
-    .report-instance-table td::before {
-        float: left;
+    .label {
+        // This functions as the width of the label; it is sized to match the widest
+        // text we know the label to render in actual usage (currently "How to check"),
+        // plus 4px of fudge-factor to account for differences in fonts across browsers/OS
+        flex-basis: 90px;
+        flex-shrink: 1;
+        flex-grow: 0;
+
+        font-size: 14px;
+        line-height: 20px;
+        font-family: $semiBoldFontFamily;
+        color: $primary-text;
+        text-align: left;
     }
 
-    .report-instance-table .label {
-        padding: 14px 20px 6px 20px;
+    .instance-list-row-content {
+        flex-basis: 200px;
+        flex-shrink: 1;
+        flex-grow: 1;
+
+        color: $secondary-text;
+        font-size: 14px;
+        line-height: 20px;
+        align-items: flex-end;
+        display: flex;
     }
 
-    .report-instance-table .instance-list-row-content {
-        padding: 6px 20px 14px 20px;
+    .instance-list-row-content.content-snipppet {
```

Comment:
I know this style was from before this change but figured it might be nice to fix the typo here.

```.content-snipppet``` should be ```.content-snippet```

Reply:
Fixed (and moved to a more specific module for the one use site)